### ISSUE TITLE
Breadcrumb error

### ISSUE THREAD

**Issue by ISC3141** [sulu/sulu#3324]
| Bug? | yes?
| New Feature? | no
| Sulu Version | 1.4.12
| Browser Version | Chrome 57.0.2987.133

When I add "{{ sulu_breadcrumb(uuid) }}" in my "master.html.twig" template to display the breadcrumb, I have the following error: An exception has been thrown during the rendering of a template ("Notice: Array to string conversion").

Do you know this issue an do you have any solutions?

Thanks in advance for your help.

**Comment by alexander-schranz** [comment]
@ISC3141 the breadcrumb is an array like the navigation.

```twig
{% for item in sulu_breadcrumb(uuid) %}
    <a  href="{{ sulu_content_path(item.url) }}">{{ item.title }}</a>
{% endfor %}
```

**Comment by ISC3141** [comment]
Thank you.
The following documentation page needs an update with this example: http://docs.sulu.io/en/latest/reference/twig-extensions/functions/sulu_breadcrumb.html

**Comment by alexander-schranz** [comment]
@ISC3141 oh thank you for report did create a PR to fix the doc example sulu/sulu-docs#323.
